website/styles/pages/admin.css

```
@@ -0,0 +1,302 @@
+.error-state i {
+    color: var(--error);
+}
+
+/* Modal Styles */
+.modal-overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background: rgba(0, 0, 0, 0.5);
```

Stylelint (17.2.0)

[error] Expected "0.5" to be "50%" (alpha-value-notation)

(alpha-value-notation)